src/domain/flow-system/src/entities/shared/schedule.rs

```diff
@@ -36,12 +40,52 @@ pub struct ScheduleCronExpression {
 
 /////////////////////////////////////////////////////////////////////////////////////////
 
+#[derive(Error, Debug)]
+pub enum ScheduleError {
+    #[error(transparent)]
+    InvalidCronExptression(#[from] InvalidCronExptressionError),
+
+    #[error(transparent)]
+    CronExpressionIterationExceed(#[from] CronExpressionIterationError),
+}
+
+#[derive(Error, Debug)]
+#[error("Cron expression {expression} is invalid")]
+pub struct InvalidCronExptressionError {
+    pub expression: String,
+}
+
+#[derive(Error, Debug)]
+#[error("Cron expression {expression} iteration has been exceed")]
+pub struct CronExpressionIterationError {
+    pub expression: String,
+}
```

Comment:
Try to add some tests to activate these errors. Perhaps, a good place is the GraphQL test suite for the API that configures flow schedules.

Reply:
Yeah, I added such a test, but only now I realize that it will not trigger during GraphQL setup. Moving it to a higher level right now.